cluster/gce/trusty/configure-helper.sh

```diff
@@ -612,6 +613,21 @@ start_kube_apiserver() {
     params="${params} --etcd-quorum-read=${ETCD_QUORUM_READ}"
   fi
 
+  if [[ "${ENABLE_APISERVER_BASIC_AUDIT:-}" == "true" ]]; then
+    # We currently only support enabling with a fixed path and with built-in log
+    # rotation "disabled" (large value) so it behaves like kube-apiserver.log.
+    # External log rotation should be set up the same as for kube-apiserver.log.
+    params="${params} --audit-log-path=/var/log/kube-apiserver-audit.log"
```

Comment:
I like `+=` better too, but I wanted to follow the style used within each file so it's self-consistent.

I'm actually thinking about reverting my change to the `trusty` and `container-linux` versions because I don't know how log rotation works on those and updating those other flavors is beyond the scope of what I'm trying to accomplish (getting this working for GCE w/ GCI).